repair/repair.cc

```diff
@@ -1600,6 +1617,19 @@ static int do_repair_start(seastar::sharded<database>& db, seastar::sharded<netw
         throw std::runtime_error("Cannot combine data centers and hosts options.");
     }
 
+    if (!options.ignore_nodes.empty() && !options.hosts.empty()) {
+        throw std::runtime_error("Cannot combine ignore_nodes and hosts options.");
+    }
+    std::unordered_set<gms::inet_address> ignore_nodes;
+    for (const auto& n: options.ignore_nodes) {
+        try {
+            auto node = gms::inet_address(n);
+            ignore_nodes.insert(node);
+        } catch(...) {
```

Comment:
What exception are you expecting here, `std::invalid_argument` resulting form parsing errors? 

Better catch it explicitly, so that we don't misinterpret other exceptions like bad_alloc.

Reply:
Also, the error message could is confusing. It says "Unknown node", but in fact the problem is different, the name of the node is not a parseable inet address.

Reply:
The error message follows the hosts option. I made it more clear in v2.  

We do not have to catch std::invalid_argument, we can just propagate the error.